dev-client/src/translations/en.json

```diff
@@ -514,6 +517,111 @@
         "VOLUME_35_60": "35-60%",
         "VOLUME_60": ">60%"
       }
+    },
+    "color": {
+      "title": "Soil Color",
+      "info": {
+        "p1": "Soil color can tell you a lot about soil, including:",
+        "bullet1": "Which minerals are present",
+        "bullet2": "How much organic matter is present",
+        "bullet3": "And much more",
+        "p2": "Soil color can be measured using a Munsell book or spectrometer.",
+        "p3": "In this app, you can also take a photo with a reference card like a Post-It Note to determine your soil color."
```

Comment:
```suggestion
        "p3": "In this app, you can also take a photo with a reference card like a Post-it note to determine your soil color."
```

Reply:
i think we are using `Post-It Note` because the user needs to use a proper name-brand post it note for the color matching algorithm to work. but i think feedback on copy belongs somewhere other than a PR, i.e. the design has already been approved with this copy, & we generally iterate on copy in the figma

Reply:
The "I" in it is not capitalized on the official 3M website. The "n" in "note" can go either way.